sdk/monitor/azure-monitor-query/src/main/java/module-info.java

```diff
@@ -9,7 +9,9 @@
     requires transitive com.azure.core.experimental;
     exports com.azure.monitor.query;
     exports com.azure.monitor.query.models;
-    opens com.azure.monitor.query.metrics.implementation.models to com.fasterxml.jackson.databind;
-    opens com.azure.monitor.query.log.implementation.models to com.fasterxml.jackson.databind;
-    opens com.azure.monitor.query.models to com.fasterxml.jackson.databind;
+    opens com.azure.monitor.query.log.implementation.models to com.fasterxml.jackson.databind, com.azure.core;
```

Comment:
> com.azure.core; [](http://example.com/codeflow?start=94&length=16)

So you needed both - yay!! please tell me why you needed it.

Reply:
Yeah, this is because I added more fields to query result type (statistics) which was reflectively accessed by azure-core and also I included a method to convert a table row to an object using JacksonAdapter.